src/components/probeDetails.jsx

```diff
@@ -0,0 +1,380 @@
+          <br />
+          <br />
+          <table className="table table-sm table-striped table-hover table-bordered border-0">
+            <tbody>
+              <tr>
+                <td className="fit pr-2">Type:</td>
+                <td id="detail-probe-type" className="grow">
+                  <a href={getProbeDocumentationURI(probe.type)}>{probe.type}</a>
+                </td>
+              </tr>
+              <tr title="Whether this probe collected on Firefox release or only on prerelease channels.">
+                <td className="fit pr-2">Population:</td>
+                <td id="detail-recording-type" className="grow">{populationLabel}</td>
+              </tr>
+              {getDatasetInfoRow(datasetInfo)}
+            </tbody>
+          </table>
+          <br />
+          <div id="detail-description">{probeInfo.description}</div>
+          <br />
+          <table className="table table-sm table-striped table-hover table-bordered border-0">
+            <tbody>
+              <tr>
+                <td className="fit pr-2">Bug numbers:</td>
+                <td id="detail-bug-numbers" className="grow">
+                  {bugs.map(bugNumber => {
+                    return (
+                      <React.Fragment key={bugNumber}>
+                        <a href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${bugNumber}`}>bug {bugNumber}</a>,{' '}
+                      </React.Fragment>
+                    );
+                  })}
+                </td>
+              </tr>
+              <tr title="What versions this probe is actually recorded in. This depends on when the probe was added, removed and its expiry.">
+                <td className="fit pr-2">Recorded in versions:</td>
+                <td id="detail-recording-range" className="grow">
+                  {getInfoList(rangeText)}
+                </td>
+              </tr>
+              <tr title="The probe will automatically expire in and stop recording in this version. This means that the probe will record at most until the version before that. Note that the code recording it could be removed before that.">
+                <td className="fit pr-2">Expiry:</td>
+                <td id="detail-expiry" className="grow">{getInfoList(expiryText)}</td>
+              </tr>
+              {probeInfo.cpp_guard &&
+                (<tr>
+                  <td className="fit pr-2">Preprocessor guard:</td>
+                  <td className="grow">{probeInfo.cpp_guard}</td>
+                </tr>)
+              }
+              {getExtraProbeDetails(probe, probeInfo).map(detail => {
+                return (
+                  <tr key={detail.label}>
+                    <td className="fit pr-2">{detail.label}:</td>
+                    <td className="grow">{detail.content}</td>
+                  </tr>
+                );
+              })}
+            </tbody>
+          </table>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default ProbeDetails;
```

Comment:
Do you think it would be over-optimizing to split this file up? It's pretty big, but I can be convinced otherwise.